source/_integrations/roon.markdown

```diff
@@ -28,3 +28,31 @@ You need the Hostname or IP address of the machine that runs your Roon Core. Thi
 3. Home Assistant will then contact your Roon Core and ask to be authorized. You will need to enable this extension in the Room Application. Go to **Settings** and then **Extensions**, there you will see an entry for Home Assistant with a button next to it. Click **Enable**.
 4. Roon core will then provide Home Assistant with the details of your media players.
 5. In Home Assistant you can then pick an area for each of your music players, and add them to Home Assistant.
+
+### Service `join`
+
+Group players together.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | id of the player that will be the master of the group.
+| `join_ids` | no | id(s) of the players that will join the master.
+
+### Service `unjoin`
+
+Remove players from a group.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | id of the player that is the master of the group.
+| `unlink_name` | yes | id(s) of the players that will be unjoined from the group. If not specified, all players will be unjoined from the master.
+
+
+### Service `transfer`
+
+Transfer playback from one player to another.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | id of the source player.
```

Comment:
This service is registered using `async_register_entity_service`, which means the `entity_id` is optional (as a group of entities in an area can be targeted alternatively).

```suggestion
| `entity_id` | yes | id of the source player.
```